src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import "./variables";
+
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  width: $column-width;
```

Comment:
The width of the .calendar block should not be equal to $column-width, it should be calculated based on the number of columns, the width of the day cell, and the gaps between them. The correct calculation should look like this: width: $day-width * 7 + $gap * 6;